src/clr_bundle_ls.c

```diff
@@ -133,7 +138,20 @@ static bool parse_options(int argc, char **argv)
 			}
 			set_cert_path(optarg);
 			break;
-
+		case 'D':
+			if (!optarg) {
+				fprintf(stderr, "Invalid --has-dep argument\n\n");
+				goto err;
+			}
+			cmdline_option_has_dep = optarg;
```

Comment:
Use `string_or_die()` here.

Reply:
I am suggesting this primarily for consistency; the rest of the code base only assigns dynamically allocated strings to the globals.

Reply:
I agree! Fixed.